src/util/util.js

```diff
@@ -47,7 +47,18 @@ function randIndex(array = []) {
   return array[Math.floor(Math.random() * array.length)];
 }
 
+function capitalize(s) {
+  return s[0].toUpperCase() + s.slice(1);
+}
+
 module.exports = {
+  randInt: randInt,

```

Comment:
if you use the same names in a object it's possible to do it like this

  ```
randInt,
  randNumber,
  randNumberBias,
```
  etc...

Not sure if it will work for your situation, but it will reduce the amount of duplication you have here :)